docs/10-cryptography/01-symmetric-key-cryptography.md

```diff
@@ -0,0 +1,172 @@
+---
+nav_order: 1
+title: Criptare cu cheie simetrica
+parent: Criptare și semnarea datelor
+---
+
+<details markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+# Criptare cu cheie simetrica
+
+Criptarea este procesul de transformare a datelor într-o formă ilegibilă sau neinteligibilă pentru a proteja informațiile în timpul transmiterii sau stocării acestora. Scopul principal al criptării este să se asigure că doar persoanele autorizate, care dețin cheia de decriptare corespunzătoare, pot accesa și înțelege informațiile.
+
+Există două tipuri principale de criptare:
+
+1. **Criptare cu cheie simetrică:**
+   - Utilizează aceeași cheie pentru atât criptare, cât și decriptare.
+   - Este eficientă și rapidă, dar implică provocări legate de securitatea distribuirii cheii între părți.
```

Comment:
Extinde puțin idea asta: dificultatea distribuirii între părți aflate la distanțe mari / online